index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1 class="header">
+        Hello Microverse!
+    </h1>
+    
```

Comment:
- [ ] Kindly changes your readme as the reviewer said in the previous review. Remember that you need to make these changes on your actual branch, not on the main.

Reply:
@BregornOriginal i think adding my readme file in main is okay because at the end will merge